composeApp/build.gradle.kts

```diff
@@ -81,6 +81,10 @@ kotlin {
             implementation(libs.bundles.ui)
             implementation(libs.bundles.tooling)
 
+            implementation(libs.androidx.datastore.preferences.core)
+            implementation(libs.androidx.datastore.core.okio)
+            implementation(libs.kotlinx.atomicfu)
```

Comment:
I feel we can use more bundles, to avoid having to add here each individual dependency.